activerecord/lib/active_record/connection_adapters/abstract/transaction.rb

```diff
@@ -125,6 +125,7 @@ def after_commit; yield; end
       def after_rollback; end
       def user_transaction; ActiveRecord::Transaction::NULL_TRANSACTION; end
       def isolation=(_); end
+      def fixture_transaction?; false; end
```

Comment:
I'm unsure about this bit. If we say that `fixture_transaction? == !joinable` then this should be `true`.